proposals/hackygolucky-EXPECTATIONS/EXPECTATIONS.md

```diff
@@ -0,0 +1,108 @@
+## Expectations
+
+With the intent to merge the Node.js Foundation and JS Foundation being announced, those interested in the continued support of Node.js and JS-related projects are working through expectations and structure for what the potential new foundation could look like.
+
+This document is for the initial phase of work for the Bootstrap team to draft requirements that overlap with scope of the Board in order to provide an outline and structure for the foundation attuned to the needs of the projects and ecosystem stakeholders who have participated.
+
+This document is organized by two key areas: concerns that must be included in the foundation bylaws, and those that are communicated expectations that aren't required to be in our legal documents to proceed but are vital to managing expectations of project members and ecosystem stakeholders.
+
+### Bylaws requirements
+- Communicated scope for the foundation and board 
+- Annual reviews of staff, NPS, board, programs, support services to projects for accountability and incremental improvement 
+- The foundation staff, executive director, and board should follow a leadership style that prioritizes and promotes the well-being of the project and those who prioritize and promote a healthy project and ecosystem. 
+- Project representation on the Board 
+    - Community Representation should sit on the Board directly. Projects could share this representation. 
+    - Community representation on the Board: 3 seats guaranteed for community representation 
+- Board must establish Cross Project Council focused on the specific needs of the member projects 
+    - Allow projects to self-organize amongst themselves. Processes are defined and driven by project e.g. CPC is not structured by the Board 
```

Comment:
What are the responsibilities of the CPC?

Reply:
Currently being hashed out in issue https://github.com/nodejs/bootstrap/issues/37

This document doesn't have to outline full governance of these things, but the overarching expectation that these things MUST exist in order for us to succeed. Minimal additions to the ByLaws.